### PR TITLE
HRINT-2764 - license updates

### PR DESCRIPTION
**Pull request by grisha-kotler**
### Additional description

License updates

### How risky is the change?

- Low 

### Testing 

- It has been verified by manual testing

### CODE REVIEW

File: src/Raven.Server/Commercial/LicenseStatus.cs [hunk 1]
```diff
@@ -60,6 +60,12 @@ private T GetValue<T>(string attributeName)
 
         public DateTime? Expiration => GetValue<DateTime?>("expiration");
 
+        public bool CanActivate(out DateTime? canBeActivateUntil)
+        {
+            canBeActivateUntil = GetValue<DateTime?>("canBeActivatedUntil");
+            return canBeActivateUntil == null || canBeActivateUntil.Value < DateTime.UtcNow;
```
Comment (ayende) [inline src/Raven.Server/Commercial/LicenseStatus.cs]:
```suggestion
            return canBeActivateUntil == null || canBeActivateUntil.Value <= DateTime.Today;
```

Comment (grisha-kotler) [reply src/Raven.Server/Commercial/LicenseStatus.cs]: done

File: src/Raven.Server/Commercial/LicenseStatus.cs [hunk 2]
```diff
@@ -60,6 +60,12 @@ private T GetValue<T>(string attributeName)
 
         public DateTime? Expiration => GetValue<DateTime?>("expiration");
 
+        public bool CanActivate(out DateTime? canBeActivateUntil)
+        {
+            canBeActivateUntil = GetValue<DateTime?>("canBeActivatedUntil");
+            return canBeActivateUntil == null || canBeActivateUntil.Value >= DateTime.Today;
```
Comment (ppekrol) [inline src/Raven.Server/Commercial/LicenseStatus.cs]: DateTime.UtcNow.Date instead of DateTime.Today ?

Comment (ayende) [reply src/Raven.Server/Commercial/LicenseStatus.cs]: Probably best, yes

**Pull request merged**